src/pages/documentation/adobe-campaign-standard/index.md

```diff
@@ -189,6 +189,25 @@ iOS
 
 Giving a value of `0` when setting `campaign.registrationDelay` will send a registration request on every launch event. This is the previous behavior seen before the registration request reduction enhancement was added.
 
+### Using a bundled asset as a fullscreen in-app message fallback image
+
+A bunded image asset may be specified on the Campaign Standard UI to be used as a primary image or as a fallback image in the case where a specified remote image url is inaccessible. The bundled image should be specified on the Campaign Standard UI with the file name and file extension. For example, in the `Media URL` or `Bundled Image` text entry field on the Campaign Standard UI, a bundled JPEG file with the file name `adobe` can be provided in the following format:
+
+```text
+adobe.jpg
+```
+
+The specified bundled image must then be included with your app when it is built. To do so:
+
+#### Android
```

Comment:
Should we use Tabs block here?

Reply:
Actually yeah, a tab _would_ be a good use here. Let's do that instead.

Reply:
sure, will update this